### PR TITLE
(CDAP-5457) Added documentation for collapsing lineage output.

### PR DESCRIPTION
**Pull request by bdmogal**
Jira: [CDAP-5457](https://issues.cask.co/browse/CDAP-5457)
Quick Build: http://builds.cask.co/browse/CDAP-DOB196
Passes [Quick Build 7](http://builds.cask.co/browse/CDAP-DOB196-7)
Relevant Doc Pages:
[Lineage RESTful API](http://builds.cask.co/artifact/CDAP-DOB196/shared/build-7/Docs-HTML/3.4.0-SNAPSHOT/en/reference-manual/http-restful-api/metadata.html#http-restful-api-metadata-lineage)
[Collapsing Lineage Output Section](http://builds.cask.co/artifact/CDAP-DOB196/shared/build-7/Docs-HTML/3.4.0-SNAPSHOT/en/reference-manual/http-restful-api/metadata.html#lineage-collapse)

### CODE REVIEW

**Comment by awholegunch** [comment]
Please review my commit. Started a new quick build.

File: cdap-docs/reference-manual/source/http-restful-api/metadata.rst [hunk 1]
```diff
@@ -663,14 +663,15 @@ where:
      - Number of levels of lineage output to return. Defaults to 10. Determines how far back the provenance
        of the data in the lineage chain is calculated.
    * - ``<collapse>``
-     - An optional set of `collapse` types to collapse the lineage output by. By default, lineage output is
-       not collapsed.
+     - An optional set of ``collapse`` types by which to 
+       :ref:`collapse the lineage output <http-restful-api-metadata-lineage-collapse>.
```
Comment (bdmogal) [inline cdap-docs/reference-manual/source/http-restful-api/metadata.rst]: somehow this linking did not work for me. Will check the output of the quick build.

**Comment by bdmogal** [comment]
Thanks @awholegunch. Changes look good. I will wait for @poornachandra to sign off on the content.

**Comment by poornachandra** [comment]
LGTM

**Comment by bdmogal** [comment]
Thanks @awholegunch @poornachandra. Squashed commits. Merging.

**Pull request merged**